README.md

```diff
@@ -1,16 +1,46 @@
-# MultCo API
+# Multco API
 
-## Authorization
-Developers may turn authorization on/off by performing the following steps in Visual Studio:
-- Go to **project** > **properties**
-- Select desired configuration (e.g. Debug/Release)
-- In the "Conditional compilation symbols" field, define a variable `AUTH`
-- Make sure to save the changes
-- Build project
```

Comment:
If you have time I think it'd still be useful to keep this. But we can merge without.